src/main/java/org/fundacionjala/pivotal/objects/CreateProject.java

```diff
@@ -0,0 +1,136 @@
+
+    @FindBy(how = How.CSS, using = "#view42 img.headerLogo__image")
+    private WebElement goToDashBoard;
+
+    @FindBy(how = How.XPATH, using = "//*[@id=\"modal_area\"]/div/div[2]/div/form/footer/button[2]")
+    private WebElement buttonCreate;
+
+    @FindBy(how = How.CSS, using = "#modal_area  span.tc-account-selector__create-account-text")
+    private WebElement buttonCreateAccount;
+
+    @FindBy(how = How.CSS, using = "#modal_area  input.tc-account-creator__name")
+    private WebElement txtAccountName;
+
+    /**
+     * Set go to DashBoard.
+     */
+    public void setGoToDashBoard() {
+        driverWait.until(ExpectedConditions.visibilityOf(goToDashBoard));
+        goToDashBoard.click();
+    }
+
+    /**
+     * Method for verifyName.
+     * @return txtVerifyName.
+     */
+    public String verifyName() {
+        waitWebElement(txtVerifyName);
+        return txtVerifyName.getText();
+    }
+
+    /**
+     * Method for set Privacy.
+     * @param privacy String.
+     */
+    private void setPrivacy(final String privacy) {
+        if (privacy.equalsIgnoreCase("private")) {
+            driver.findElement(By.cssSelector(
+                    "div#modal_area label:nth-child(2) > input[name=\"project_type\"]")).click();
+        }
+        if (privacy.equalsIgnoreCase("public")) {
+            driver.findElement(By.cssSelector(
+                    "div#modal_area label:nth-child(3) > input[name=\"project_type\"]")).click();
+        }
+    }
+
+    /**
+     * Method for set Select an Account Specific.
+     * @param account String.
+     */
+    private void setTxtSelectorAcccountSpecific(final String account) {
+        this.setTxtSelectorAcccountCheck();
+
+        //these last comands permit to create a new Account from create project menu
+        CommonMethods.clickWebElement(buttonCreateAccount);
+        CommonMethods.setWebElement(txtAccountName, account);
+    }
+
+    /**
+     * Method for Select Account Check.
+     */
+    private void setTxtSelectorAcccountCheck() {
+        CommonMethods.clickWebElement(txtSelectorAcccountCheck);
+    }
+
+    /**
+     * Method for set Button Create Project.
+     */
+    public void setButtonCreateProject() {
+        CommonMethods.clickWebElement(buttonCreateProject);
+    }
+
+    /**
+     * Method for set Project name.
+     * @param projectName String.
+     */
+    private void setTxtProjectName(final String projectName) {
+        CommonMethods.setWebElement(txtProjectName, projectName);
+
+    }
+
+    /**
+     * Method for set Button Create.
+     */
+    public void setButtonCreate() {
+        CommonMethods.clickWebElement(buttonCreate);
+    }
+
+    /**
+     * Method for Strategy Step Map.
+     * @param values Map<ProjectValues, Object>
+     * @return strategyMap.
+     */
+    public final Map<ProjectValues, Meth> getStrategyStepMap(final Map<ProjectValues, Object> values) {
+        Map<ProjectValues, Meth> strategyMap = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)